### ISSUE TITLE
(2.x) is there any way to kill a custom setup after it has initialized on my form?

### ISSUE THREAD

**Issue by jasonfb** [braintree/braintree-web#204]
with a very basic 'custom' setup, I have a simple form working with the 2.27.0 version of this library here:

https://gist.github.com/jasonfb/73a769ade9e81f29c512038fea95c7e2

Can I turn the braintree setup off entirely after it has been initialized? My normal first-card entry workflow works fine, but I want the user to be able to select an existing credit card and this is where I am getting stuck because the Braintree javascript is throwing "User did not input payment method" validation on the form

So basically if the user chooses from an existing card instead of entering a new one I want the previously-initialized braintree to be disabled. 

Something like 

braintree.kill()

Anything available like this in the 2.27.0 version?

**Comment by crookedneighbor** [comment]
There sure is! It's called `teardown`. Here are the docs for it: https://developers.braintreepayments.com/reference/client-reference/javascript/v2/best-practices#teardown
